metta/sim/simulation.py

```
@@ -174,8 +165,13 @@ def create(
         run_name: str = "simulation_run",
     ) -> "Simulation":
         """Create a Simulation with sensible defaults."""
-        # Get policy record or create a mock
-        policy_record = policy_store.policy_record_or_mock(policy_uri, run_name)
+        # Create policy record from URI
+        if policy_uri:
```

we probably want to fail loudly if your policy isnt found?